microsoft-edge/devtools-guide-chromium/source-maps-monitor/source-maps-monitor-tool.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Source Maps Monitor tool
+description: How to use the Source Maps Monitor tool in Microsoft Edge DevTools.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 02/15/2022
+---
+# Source Maps Monitor tool
+
+Use the **Source Maps Monitor** tool to monitor your source maps.  This tool works together with source maps, to [Map preprocessed code to source code](../javascript/source-maps.md).
+
+To try this feature:
+
+1. In a new tab or window, follow the instructions in [Map preprocessed code to source code](../javascript/source-maps.md).
+
+1. Close the popup dialog box that the demo opens.
+
+1. In the **Sources** tool, in the **Page** tab, select the file that the demo created for you, such as **Coffee2.js**.
+
+1. In DevTools, on the main toolbar, click the **More Tools** (![More Tools icon.](../media/more-tools-icon-light-theme.png)) button, and then select the **Source Maps Monitor** tool.
+
+   ![The Source Maps Monitor tool.](../media/source-maps-monitor-tool.png)
+
+   _Note: To see details of the above image, right-click it, and then select **Open image in new tab**.  Close the image tab to return to the article._
```

Comment:
I don't think we should have this note here. It feels odd having it here but not for the other ones. And I wouldn't want to clutter our docs with notes about how to zoom images next to each and every one of them. So I'd tend to just remove this note and rely on 2 other things: 1. let people learn/know how to zoom on their own and 2. improve d.m.c to provide a way to zoom even markdown images without having to use `:::` lightbox images.

Reply:
@captainbrosset 
Glad you pointed out this early version of the tip.  I changed it to the latest tagging:

_To zoom: right-click > Open image in new tab._

Pattern of applying that tip: 
In each h2 section that contains a wide, detailed image, the tip is below the image that needs it the most.
Test the page with full- & half-width window.

Find "to zoom" 6x in 30-page article:
https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/accessibility/accessibility-testing-in-devtools?branch=pr-en-us-1665

html tagging in .md file might be able to refine styling eg make smaller to offset the tip from body text.

Reply:
@captainbrosset - also see new h2 in new article in present PR
https://review.docs.microsoft.com/en-us/microsoft-edge/develop-web-microsoft-edge?branch=pr-en-us-1665#zoom-an-image-in-the-docs
Similar to "let people learn/know how to zoom on their own ... improve d.m.c to provide a way to zoom even markdown images without having to use ::: lightbox images."

Reply:
I'm still not sure we need this tip. 2 reasons come to mind, in addition to what I mentioned earlier:

1. The string "Open image in new tab" might not always be the same in all browsers on all devices. In particular on mobile, the right-click context menu does not exist. And I'm not sure that even on desktop all browsers have the same UI string.
2. At half-width, the DMC site hides the sidebar and presents the article full-width. When that happens, right-clicking to open the image in a new tab does not, actually, zoom the image at all. The image is already almost full width, so opening it in a new tab does not change its size.